docs/xml/enumerations/vote_variation.rst

```diff
@@ -1,35 +1,32 @@
 VoteVariation
 =============
 
-.. todo::
-
-   Document VoteVariation
-
+    
 +----------------------+----------------------------------------------------------------------------------+
 | Name                 | Description                                                                      |
 |                      |                                                                                  |
 +======================+==================================================================================+
-| 1-of-m               |                                                                                  |
-+----------------------+----------------------------------------------------------------------------------+
-| approval             |                                                                                  |
+| 1-of-m               | When a voter can select only one candidate in a contest.                         |
 +----------------------+----------------------------------------------------------------------------------+
-| cumulative           |                                                                                  |
+| approval             | When a voter can select as many candidates as desired.                           |
```

Comment:
Personally I think saying that the "approval" value of `VoteVariationType` means "Approval voting" is redundant, unless the "Approval voting" text is a hyperlink to wikipedia. I'll welcome other comments on this one, though.